java/openmldb-taskmanager/src/main/java/com/_4paradigm/openmldb/taskmanager/config/TaskManagerConfig.java

```diff
@@ -68,7 +69,10 @@
             ZK_MAX_CONNECT_WAIT_TIME = Integer.parseInt(prop.getProperty("zookeeper.max_connect_waitTime", "30000"));
             OFFLINE_DATA_PREFIX = prop.getProperty("offline.data.prefix");
             SPARK_MASTER = prop.getProperty("spark.master", "yarn");
-            BATCHJOB_JAR_PATH = prop.getProperty("batchjob.jar.path");
+            BATCHJOB_JAR_PATH = prop.getProperty("batchjob.jar.path", "");
+            if (BATCHJOB_JAR_PATH.isEmpty()) {
+                BATCHJOB_JAR_PATH = BatchJobUtil.findLocalBatchJobJar();
```

Comment:
need test coverage

Reply:
We will add check for config of TaskManager in this issue https://github.com/4paradigm/OpenMLDB/issues/1252 .